src/app/Rules/Cnp.php

```diff
@@ -26,9 +26,9 @@ public function __construct()
      */
     public function passes($attribute, $value)
     {
-        
-        $exist = Volunteer::query()->where('ssn', '=', $value)->first();
-        if (strlen($value) === 13 && !$exist) {
+//        $exist = Volunteer::query()->where('ssn', '=', $value)->first();
+//        if (strlen($value) === 13 && !$exist) {
```

Comment:
Please delete the lines that are no longer needed.

Reply:
ok. I will remove